pkg/config/config_template.yaml

```diff
@@ -45,6 +45,21 @@ api_key:
 #   - env:prod
 #   - role:database
 
+# Split tags according to given string.
+# Supported only for tags handled by the Tagger (Kubernetes, Docker, AWS ECS, ...)
```

Comment:
This comment probably needs to be updated. Seems like now the scope is:
  - container tags
  - host tags
and it excludes
  - tags collected by integrations themselves

Reply:
👍 good point, I just updated the messaging.